wg-advisors/why/why_discussions_and_decisions.md

```diff
@@ -1,13 +1,77 @@
 
-TODO:
-
 ## Summary 
 
+Discussions and decision making should happen on the mailing list to ensure that the project is
+transparent and that all decisions are made in the open. This is important to ensure that all
+community members have the opportunity to participate in the decision making process.
+
 ## Links to relevant documents
 
+* [The Apache Way](https://www.apache.org/theapacheway/)
+* [Mailing list guidelines](https://www.apache.org/foundation/mailinglists.html)
+
 ## Why are we doing it?
 
+Mailing list the primary communication channel for the project. It has been chosen - at the dawn of the
+Foundation as the primary communication channel because it was the most inclusive and accessible channel
+for all community members, allowing for asynchronous communication and ensuring that all community members
+have the opportunity to participate. All the important decisions and discussions happens there - including
+the decision to invite new committers and PMC members, the decisions to release software, and the decisions
+to change the project's direction.
+
+Why the time passed and technology evolved, the mailing list still keep a lot of the properties that made it
+the primary communication channel for the project. It is still the most inclusive and accessible channel for
+all community members, they allow for asynchronous communication and do not require from the community members
+to be online at the same time and to setup accounts with particular external services.
+
+They also provide a permanent record of the discussions and decisions and the Apache Software Foundation
+Infrastructure team makes sure that the archives are preserved and available for the future, including the
+ability to search and browse the archives, as well as permanently link to the discussions and decisions so
+that they can be always referred to in the future.
+
+The communication on mailing list happens in English, which is the language of the Foundation,
+and the language of the majority of the Foundation's community members. 
+
+An important aspect of using the mailing list is that it allows anyone (including PMC members, committers,
+contributors, and users, but also people from outside - Board members, other PMCs, non-ASF folks) to take
+a look at the way how decisions are made and even participate in the discussions and decisions, and to
+peek into the PMC status. This is important for the Foundation's transparency and for the community building
+process.
+
+PMCs have at least two lists: dev@ and private@ - but it is important to mention that the private@ list should
+only be used for discussions that are absolutely not suitable for public consumption, such as discussions
+about security vulnerabilities, or discussions about individuals that are not suitable for public consumption
+and should be kept private. The private@ list should not be used for discussions about the project's direction,
+releases, or other important decisions.
```

Comment:
One possible exception is discussing/voting on a release with a security issue.

Reply:
Yep. Added.